ydb/core/tx/datashard/datashard_ut_write.cpp

```diff
@@ -147,7 +145,7 @@ Y_UNIT_TEST_SUITE(DataShardWrite) {
 
         Cout << "========= Read table with 1th row deleted =========\n";
         {
-            auto tableState = TReadTableState(server, MakeReadTableSettings("/Root/table-1")).All();
+            auto tableState = ReadTable(server, shards, "table-1", tableId, 2000);
```

Comment:
Please, hide the magic number 2000. It has no semantic meaning for tests like this.

Reply:
We need to distinguish `ReadTable` calls in one test to generate a unique `readId`. Maybe make parameter like`readOrder` in `ReadTable` and multiply by 1000 and increment inside `ReadTable`? Alternatively, make a test helper class that holds and tracks `readId`?

Reply:
Let's do `ui64 startReadId = 1000` as suggestion here https://github.com/ydb-platform/ydb/pull/7121#discussion_r1693921476

Reply:
So, I'll add a default value for the `startReadId` parameter and leave this non-default 2000 because there are multiple `ReadTable` calls in this particular test (and do so for other simialr), is that correct?

Reply:
Yes, let's do it.